srcpkgs/conky/template

```diff
@@ -1,25 +1,34 @@
 # Template file for 'conky'
 # Keep in sync with 'conky-cli'
 pkgname=conky
-version=1.13.1
+version=1.15.0
 revision=1
+archs="~i686*"
 build_style=cmake
 conf_files="/etc/conky/conky.conf /etc/conky/conky_no_x11.conf"
 configure_args="-DMAINTAINER_MODE=ON -DRELEASE=ON
  -DDOC_PATH=share/doc/${pkgname} -DBUILD_X11=ON -DBUILD_CURL=ON -DBUILD_XDBE=ON
  -DBUILD_RSS=ON -DBUILD_WEATHER_METAR=ON -DBUILD_IMLIB2=ON -DBUILD_WLAN=ON
  -DBUILD_LUA_CAIRO=ON -DBUILD_LUA_IMLIB2=ON -DBUILD_LUA_RSVG=ON
  -DBUILD_XSHAPE=ON -DBUILD_DOCS=ON"
-hostmakedepends="libtool pkg-config docbook2x man-db toluapp"
+hostmakedepends="python3 python3-yaml python3-Jinja2
+ libtool pkg-config docbook2x man-db toluapp $(vopt_if man pandoc)"
 makedepends="alsa-lib-devel imlib2-devel libXdamage-devel libXft-devel
  libXinerama-devel libcurl-devel librsvg-devel lua53-devel ncurses-devel
  toluapp-devel wireless_tools-devel"
 short_desc="Lightweight system monitor for X"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="DragonGhost7 <darkiridiumghost@gmail.com>"
 license="BSD-3-Clause, GPL-3.0-or-later"
 homepage="https://github.com/brndnmtthws/conky"
 distfiles="https://github.com/brndnmtthws/conky/archive/v${version}.tar.gz"
-checksum=f85c96250b501e937b9b5d2ff3f06176e14777c94cec81373bbfc8a619d7a05a
+checksum=bebd54637bf8819ee395b765e3615ad28606d28a7870ac99ed1dffb1d14087b2
+
+build_options="man"
+desc_option_man="Use pandoc for manpages"
+
+case "$XBPS_MACHINE" in
+	x86_64*|i686|ppc64le*|ppc64) build_options_default="man" ;;
+esac
```

Comment:
or we could get the prebuilt manpage from `https://github.com/brndnmtthws/conky/releases/download/v1.15.0/conky.1.gz`

Reply:
Not building the man pages is a good idea, but originally i imagine there were 2 templates because they have different configure_args. But that can be fitted into one, just like vim, so there should be no problem there.